lesson4/recursion5.js

```diff
@@ -0,0 +1,14 @@
+//count 9('son9birt9');
+
+const count9=(a,i=0,found=0)=>{
```

Comment:
does this code apply for all other cases?

Reply:
then I have to add another parameter, right?

Reply:
done, thanks!